admin/functions-notice.php

```diff
@@ -85,4 +85,84 @@ function aioseop_notice_disable_woocommerce_detected_on_nonpro() {
 		global $aioseop_notices;
 		$aioseop_notices->deactivate_notice( 'woocommerce_detected' );
 	}
+
+	/**
+	 * Set Notice for Disabled Public Blog
+	 *
+	 * Admin Notice when "Discourage search engines from indexing this site" is
+	 * enabled in Settings > Reading.
+	 *
+	 * @since 2.4.2
+	 *
+	 * @global AIOSEOP_Notices $aioseop_notices
+	 *
+	 * @param boolean $update Updates the notice with new content and configurations.
+	 * @param boolean $reset  Notice are re-initiated.
+	 */
+	function aioseop_notice_set_blog_public_disabled( $update = false, $reset = false ) {
+		global $aioseop_notices;
+
+		$notice = aioseop_notice_blog_visibility();
+
+		if ( ! $aioseop_notices->insert_notice( $notice ) ) {
+			if ( $update ) {
+				$aioseop_notices->update_notice( $notice );
+			}
+			if ( $reset || ! isset( $aioseop_notices->active_notices[ $notice['slug'] ] ) ) {
+				$aioseop_notices->activate_notice( $notice['slug'] );
+			}
+		}
+	}
+
+	/**
+	 * Notice - Blog Visibility
+	 *
+	 * Displays when blog disables search engines from indexing.
+	 *
+	 * @since 2.4.2
+	 *
+	 * @return array
```

Comment:
a description of what it returns would help

Reply:
Added